tests/test_ecs.py

```diff
@@ -6,6 +6,31 @@
 
 class TestECS(unittest.TestCase):
 
+    def test_allow_placement_strategy_constraint(self):
+        task_definition = ecs.TaskDefinition(
+            "mytaskdef",
+            ContainerDefinitions=[
+                ecs.ContainerDefinition(
+                    Image="myimage",
+                    Memory="300",
+                    Name="mycontainer",
+                )
+            ],
+            Volumes=[
+                ecs.Volume(Name="my-vol"),
+            ],
+        )
+        ecs_service = ecs.Service(
+            'Service',
+            Cluster='cluster',
+            DesiredCount=2,
+            PlacementStategies='spread',
```

Comment:
Shouldn't this have failed?  According to https://github.com/SPSCommerce/troposphere/pull/1/files#diff-9bac6e96fe4a57c4b2bcfcb35725fa1aR39, this is supposed to be a list of PlacementStrategy instances.